modules/ROOT/pages/api-proxy-landing-page.adoc

```diff
@@ -4,123 +4,68 @@ include::_attributes.adoc[]
-
-// ==== Runtime Fabric headers
-
-// The following headers are added by Edge in Runtime Fabric:
-// [source,RAML,linenums]
-// ----
-// headers:
-//   x-correlation-id:
-//     type: string
-//   x-forwarded-for:
-//     type: string
-//   via:
-//     type: string
-//   x-forwarded-host:
-//     type: string
-// ----
-
-// Therefore, consider adding these headers in the API specification when using Edge.
+image::api-gateway-capabilities-mule4-62bf9.png[align=center]
+ 
+As shown in the diagram, when an external consumer application sends a request, it first pings the proxy endpoint. At this time, all policies applied to the API that are referenced in the proxy application takes effect. 
+
+If a policy fails any of the validations, an error response is returned (appropriate to the failed policy) and the request does not reach your backend. Conversely, if all the validations and actions performed by the policies are successful, the request continues to your backend API, and the proxy then returns this response to the consumer application.
+ 
+== API Proxy Advantages
+
+API proxies can improve the performance of your team and secure your APIs by using policies and validations. The API proxies:
+
+* Increases Development Time
++
+You can deploy a proxy directly to a xref:runtime-fabric::index.adoc[Runtime Fabric], xref:runtime-manager::cloudhub.adoc[CloudHub], xref:runtime-manager::deployment-strategies#hybrid-deployments.adoc[Hybrid] or xref:mule-runtime::mule-deployment-model.aodc[standalone Mule runtime engine] instance with just a few clicks. Even if you don't know how to create a Mule Application, API Manager builds and configures the Mule Application with an Autodiscovery so your API can be automatically tracked by API Manager after the deployment has completed.
+* Provides Flexibility for Customizations
++
+In most cases, the proxy generated in API Manager is suitable for deployment. However, because it is a Mule , you can edit it using xref:studio::import-export-packages.adoc[Anypoint Studio] to fulfill additional requirements and customization.
+* Secures and Governs your APIs Using Policies and xref:viewing-api-analytics.adoc[API Analytics]
++
+The proxy enables you to protect your API with the full capabilities of the API gateway, including access to API Analytics.
+* Implements Validations
++
+Most proxies, including RAML, REST, and WSDL proxies enable you to perform validations on all incoming requests against your API definition that you design. You can choose different levels of validation, depending on your requirements:
+** Regular validations
++ 
+Compares the payload, query parameters, URI parameters, headers, and form parameters against the schema defined in your API specification. When using this configuration, the unspecified query parameters and headers in the API specification are also sent to the backend service. 
```

Comment:
validations compare; compare with
```suggestion
Compare the payload, query parameters, URI parameters, headers, and form parameters with the schema defined in your API specification. When using this configuration, the unspecified query parameters and headers in the API specification are also sent to the backend service. 
```
Does your audience understand what you mean by "unspecified"? Is it safe to validate an unspecified value?

Reply:
Yes! Done! Thanks!